src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PackageReference.xaml

```diff
@@ -85,6 +85,11 @@
                     DisplayName="NoWarn"
                     Description="Comma-delimited list of warnings that should be suppressed for this package" />
 
+    <StringProperty Name="GeneratePathProperty" 
+                    Visible="True"
+                    DisplayName="Generate Path Property"
```

Comment:
"Generate path property" I know that PackageReference isn't consistent here, I've filed https://github.com/dotnet/project-system/issues/4231 to fix this.